tests/main_test.py

```diff
@@ -768,6 +768,17 @@ def test_config_file(tmpdir: Any, capsys: Any) -> None:
     for file in cython_files:
         assert f'{file}:1:11: E701 multiple statements on one line' in out
 
+    # .cython-lint.toml takes precedence, even if empty
+    config_file = os.path.join(tmpdir, '.cython-lint.toml')
```

Comment:
nit: it's best to write separate test functions for separate cases. Otherwise, the test report is messy and parts of a test function may not even run when there's problems.